manifests/d/Dell/DisplayManager/1.56.2110/Dell.DisplayManager.installer.yaml

```diff
@@ -0,0 +1,21 @@
+PackageIdentifier: Dell.DisplayManager
+PackageVersion: 1.56.2110
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://dl.dell.com/FOLDER09067004M/1/ddmsetup2110.exe
+  InstallerSha256: 5b868f338451be1498124f7b1363a53af9c8085081a17de488d718f4d0f2c770
+  Scope: machine
+  InstallerSwitches:
+    InstallLocation: /DIR=<INSTALLPATH>
+    Silent: /VerySilent
+  UpgradeBehavior: install
+  ProductCode: Dell Display Manager 1.56
+  AppsAndFeaturesEntries:
+  - DisplayName: Dell Display Manager 1.56
+    DisplayVersion: 1.56.2110
+    ProductCode: Dell Display Manager 1.56
```

Comment:
```suggestion
```
Apps and Features Entries must match control panel exactly if they are included. If DisplayVersion is the same as PackageVersion, it should be omitted. AppsAndFeaturesEntries are not needed for this package 

```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> Get-ARPTable

DisplayName             DisplayVersion Publisher             ProductCode                                Scope
-----------             -------------- ---------             -----------                                -----
Dell Display Manager    1.56.2110      EnTech Taiwan         {AC50C05D-9D57-40F5-B2EF-AC402F14312B}_is1 Machine
```